api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -194,7 +194,8 @@ public static void loadModules(List<File> modulesToLoad) {
 					try {
 						Module mod = loadModule(f, true); // last module loaded wins
 						log.debug("Loaded module: " + mod + " successfully");
-					} catch (Exception e) {
+					}
```

Comment:
this is not according to our style. see https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE